controllers/drplacementcontrol_controller.go

```diff
@@ -117,7 +117,6 @@ func filterMW(mw *ocmworkv1.ManifestWork) []ctrl.Request {
 
 // SetupWithManager sets up the controller with the Manager.
 func (r *DRPlacementControlReconciler) SetupWithManager(mgr ctrl.Manager) error {
-	genChangedPred := predicate.GenerationChangedPredicate{}
```

Comment:
Why did you remove this pred?  We added it because the DRPC reconciler was being called repeatedly for more than necessary iterations.

Reply:
With this pred in place, the reconciliation stops after 2 attempts. Now that we wait for cleanup (or at least reconcile till cleanup), for it to go through I had to remove the predicate.

With the addition of MW and MCV owner reference (or watches) we should be able to bring it back in.